public/locales/ne-NP/cookies-preferences.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Are we missing this one?

Reply:
No -- I'm generating these from the translation spreadsheet with some code, so it created a cookie preferences file to match that tab in the sheet. My miss on not deleting it.